codecov.yml

```diff
@@ -0,0 +1,15 @@
+# Always comment on PRs
+comment:
+  layout: "diff, flags, files" # delete old and post new
+  behavior: new
```

Comment:
should we set new?

I think is better to `update` the old one, so is cleaner and we don't have too many codecov comments... but that is just my opinion.

Reply:
What this does is delete the old comment and post a new comment. (I put the comment on the wrong line above 😆)

I like this slightly more because the new comment goes at the bottom of the PR and it can be spotted more easily, otherwise old comments get lost somewhere above in-between the commits.

Reply:
Yes you are right, is better to always have it at the bottom, I was not aware that it deletes the old comment, good to know!! like this approach more as well 👍 

Thank you for explaining!